TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/BobbyPath.java

```diff
@@ -0,0 +1,32 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+@Autonomous (group = "drive")
+public class BobbyPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new Sampl eMecanumDrive(hardwareMap);
+
+        drive.setPoseEstimate(new Pose2d(50, 50, Math.toRadians(270)));
```

Comment:
shouldn't this be 90?

Reply:
270 rotates it to the right, 90 would rotate to the left